util/chunk/chunk.go

```diff
@@ -502,6 +502,16 @@ func (r Row) GetJSON(colIdx int) json.JSON {
 	return j
 }
 
+// GetDatumRow converts chunk.Row to types.DatumRow.
```

Comment:
We should clarify that the datumRow has reference to the chunk, it's only valid when the underlying chunk is valid.

Reply:
done.